mbuild/formats/lammpsdata.py

```diff
@@ -10,18 +10,19 @@
 from mbuild import Box
 from mbuild.utils.conversion import RB_to_OPLS
 from mbuild.utils.sorting import natural_sort
+from scipy.constants import epsilon_0 # Is this okay to import?
```

Comment:
Seems okay

Reply:
`scipy` is about as stable as library as they come and it's already a dep. There's probably another way to do this but I think this is good